docs/UserGuide.adoc

```diff
@@ -85,8 +85,8 @@ and persons matching at least one keyword will be returned (i.e. `OR` search).
 
 Examples:
 
-* `find John` +
-Returns `John Doe` but not `john`.
+* `find john` +
```

Comment:
Good job on updating the user guide, however, you forgot to update the explanation of the find function above.

"The search is case sensitive" 😲 

Do take note of this.

Reply:
Will do, thank you!